Data/SentinelMissions/Sentinel5P.qmd

```diff
@@ -26,3 +26,14 @@ The main objective of the Copernicus Sentinel-5P mission is to perform atmospher
 There are [different data products](https://sentinels.copernicus.eu/web/sentinel/missions/sentinel-5p/data-products){target="_blank"} associated with the three levels of TROPOMI processing: Level-0, Level-1B and Level-2.
 
 {{< include ../_render_collections.qmd >}}
+
+:::{.callout-caution}
+
+## Processing restrictions in openEO
+
+In the present implementation of openEO and Sentinel Hub API, users are limited to processing or downloading only one band at a time when dealing with Sentinel 5P data using openEO. 
```

Comment:
drop the 'using openEO' at end of sentence, you already mention the API's to which it applies.